ArduCopter/Copter.cpp

```diff
@@ -511,30 +511,6 @@ void Copter::one_hz_loop()
     AP_Notify::flags.flying = !ap.land_complete;
 }
 
-// called at 50hz
-void Copter::update_GPS(void)
```

Comment:
Don't forget to remove the header entry for these.

Reply:
@WickedShell Done